client/app/services/blueprints-state.service.js

```diff
@@ -59,114 +59,246 @@
+      function getBlueprintPostObj(tmpBlueprint) {                                    // jshint ignore:line
+        var blueprintObj = {
+          "name": tmpBlueprint.name,
+          "description": "description",
+          "bundle": {},
+          "ui_properties": {}
+        };
+
+        if (tmpBlueprint.ui_properties && tmpBlueprint.ui_properties.chartDataModel) {
+          var chartDataModel = tmpBlueprint.ui_properties.chartDataModel;
+          if (chartDataModel.nodes) {
+            var serviceTemplates = [];
+            for (var i = 0; i < chartDataModel.nodes.length; i++) {
+              var nodeSrvTemplate = chartDataModel.nodes[i];
+              if (nodeSrvTemplate.id) {
+                serviceTemplates.push({"id": nodeSrvTemplate.id});
+              } else {
+                Notifications.warn("Cannot Save New Generic Item '" + nodeSrvTemplate.name +
+                    "'.  Saving New Generic Items Not Yet Implemented.");
+              }
+            }
+            blueprintObj.bundle.service_templates = serviceTemplates;
+          }
+          blueprintObj.ui_properties.chartDataModel = chartDataModel;
+        }
+
+        if (tmpBlueprint.content.service_catalog) {
+          if (tmpBlueprint.content.service_catalog.id !== -1) {
+            blueprintObj.bundle.service_catalog = {"id": tmpBlueprint.content.service_catalog.id};
+          } else {
+            blueprintObj.bundle.service_catalog = null;
+          }
+        }
 
-      return;
+        if (tmpBlueprint.content.service_dialog) {
+          if (tmpBlueprint.content.service_dialog.id !== -1) {
+            blueprintObj.bundle.service_dialog = {"id": tmpBlueprint.content.service_dialog.id};
+          } else {
+            blueprintObj.bundle.service_dialog = null;
+          }
+        }
+
+        var automateEntrypoints = {};
+        for (var e = 0; e < tmpBlueprint.content.automate_entrypoints.length; e++) {
+          var aep = tmpBlueprint.content.automate_entrypoints[e];
+          var newAepStr = blueprint.getEntryPointString(aep);
+          switch (aep.action) {
+            case "Provision":
+              automateEntrypoints.Provision = newAepStr;
+              break;
+            case "Reconfigure":
+              automateEntrypoints.Reconfigure = newAepStr;
+              break;
+            case "Retirement":
+              automateEntrypoints.Retirement = newAepStr;
+              break;
+          }
+        }
+
+        // console.log("Saving Entry Points: " + angular.toJson(automateEntrypoints, true));
+
+        blueprintObj.bundle.automate_entrypoints = automateEntrypoints;
+
+        blueprintObj.ui_properties.num_items = tmpBlueprint.num_items;
+
+        if (tmpBlueprint.id) {
+          blueprintObj.action = "edit";
+        }
+
+        return blueprintObj;
+      }
+
+      function updateSuccess(response) {
+        Notifications.success(__(sprintf("%s blueprint was saved.", tmpBlueprint.name)));
```

Comment:
This needs to be changed to
```ruby
Notifications.success(sprintf(__("%s blueprint was saved."), tmpBlueprint.name));
```

otherwise the string won't be extracted by `gettext-extract`